content/docs/command-reference/config.md

```diff
@@ -95,6 +95,9 @@ This is the main section with the general config options:
 - `core.checksum_jobs` - number of threads for computing file hashes. Accepts
   positive integers. The default value is `max(1, min(4, cpu_count() // 2))`.
 
+- `core.jobs` - number of workers started. Accepts positive integers. The
+  default value is `4 * cpu_count()`.
+
```

Comment:
There is no such option.

Reply:
Got it. I don't understand for now how the config sections merge and under what circumstances, thus some confusions are coming up. Thanks!